build/BuildFrameworksTasks/BuildNativeDependenciesTask.cs

```diff
@@ -25,7 +25,6 @@ public override void Run(BuildContext context)
         switch (context.Environment.Platform.Family)
         {
             case PlatformFamily.Windows:
-                configureArgs.Append("-A x64");
                 configureArgs.Append("-D CMAKE_MSVC_RUNTIME_LIBRARY=MultiThreaded");
```

Comment:
@ThomasFOG @harry-cpp this look right to you?

Reply:
Can we revert this change @arosolino , unless there is a specific reason you thought to include it?